ios-base/Extensions/ViewExtension.swift

```diff
@@ -53,6 +53,70 @@ extension UIView {
       self?.layoutIfNeeded()
     })
   }
+  
+  // MARK: Constrains Helper
+  
+  func attachHorizontally(
+    subview: UIView,
+    leadingMargin: CGFloat = UI.Defaults.margin,
+    trailingMargin: CGFloat = UI.Defaults.margin
+  ) {
+    NSLayoutConstraint.activate([
+      subview.leadingAnchor.constraint(equalTo: leadingAnchor, constant: leadingMargin),
+      subview.trailingAnchor.constraint(
+        equalTo: trailingAnchor,
+        constant: -trailingMargin
+      )
+    ])
+  }
+  
+  func addSubviews(subviews: [UIView]) {
+    for subview in subviews {
+      addSubview(subview)
+    }
+  }
+  
+  func attachVertically(
+    subview: UIView,
+    topMargin: CGFloat = UI.Defaults.margin,
+    bottomMargin: CGFloat = UI.Defaults.margin
+  ) {
+    NSLayoutConstraint.activate([
+      subview.topAnchor.constraint(equalTo: topAnchor, constant: topMargin),
+      subview.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -bottomMargin)
+    ])
+  }
+  
+  /// Center a subview already added to its parent view
+  ///
+  /// - Parameters:
+  ///   - subview: View to be centered on its parent
+  ///   - verticalOffset: Float indicating the vertical displacement of the view
+  ///   - horizontalOffset: Float indicating the horizontal displacement of the view
```

Comment:
Please use the correct type, for UIKit the values are usually CGFloat, and it is also what the function is expecting.